pageserver/src/tenant_mgr.rs

```diff
@@ -102,8 +102,7 @@ pub struct Tenant {
 /// A state of a tenant in pageserver's memory.
 #[derive(Debug, Clone, Copy, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
 pub enum TenantState {
-    // A tenant is fully prepared for operations.
-    Active,
+    Active { background_jobs_running: bool },
```

Comment:
A comment would be nice, to explain what Active means, with or without background jobs running.

Reply:
Please refresh the PR, it was added.